airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/dataflow/aggregate/AggregateStore.kt

```diff
@@ -57,36 +59,35 @@ class AggregateStore(
         return null
     }
 
-    fun removeAll(): List<Aggregate> {
-        return aggregates.values.map { it.value }
+    fun getAll(): List<AggregateEntry> {
+        return aggregates.values.toList()
```

Comment:
Nit: do we need the toList?

Reply:
It's some sort of Collection without that. Let me see